portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,7 @@
 
 // Fetches data from servelet when the document is loaded
 window.onload = async () => {
-  const response = await fetch('/data');
+  const response = await fetch('/comment');
   const jsonArray = await response.json();
```

Comment:
Surround with try/catch and handle errors. All the comments logic should also be encapsulated in its own method or create a new comments object to handle it.

Reply:
I'll fix this one in the js branch! (shouldn't include the change here, sorry)